source/core/parameters.cc

```diff
@@ -1599,6 +1600,8 @@ namespace Parameters
         variable = Variable::phase;
       if (vop == "temperature")
         variable = Variable::temperature;
+      if (vop == "velocity and temperature")
+        variable = Variable::velocity_temperature;
```

Comment:
Maybe here it would be better to move to a mask approach where we would have like 00000 means false false false false false and then 10001 for example would mean true false false false true. This would be a lot more flexible to allow multiple adaptation criteria. This mechanism proposed here won't be maintainable, every new combination of variable means a new enum variable and don't get me started if we need 3...

Reply:
I really like the idea of having a flexible approach! We can talk about how to integrate it well, also with the user input on the prm. Maybe this would be better in another PR? (and I remove the mechanism on velocity_temperature in this one?)

Reply:
I talked with Lucka and he needs to have the velocity+temperature still available, so I will implement the generalized approach directly in this PR in the end :) to be coming soon!

Reply:
Okidoke, don't hestiate to talk to me for the implementation. I Have a very good idea for the architecture.

Reply:
In the end it was becoming too big so I removed it and opening another branch for that! Lucka is ok with his case being broken for a few days. I cleaned this PR :)